File: Search Insert Position/Search_Insert_Position.cpp
class Solution {
public:
    int searchInsert(vector<int>& nums, int target) {
        int n = nums.size();
        if(nums[n-1] < target) return n;
        int ans = -1;
        for(int i =0; i<n; i++) {
            if(nums[i] >= target) {
                return i;
            }
        }
        return -1;
    }
};

File: Search Insert Position/Search_Insert_Position.java
class Solution {
    public int searchInsert(int[] nums, int target) {
        int n = nums.length;
        if (nums[n - 1] < target) return n;
        for (int i = 0; i < n; i++) {
            if (nums[i] >= target) {
                return i;
            }
        }
        return -1; 
    }
}